README.md

```diff
@@ -1,46 +1,111 @@
+
+The purpose of this project is to create *security policy content* for various
+platforms -- *Red Hat Enterprise Linux*, *Fedora*, *Ubuntu*, *Debian*, and others.
+Our aim is to make it as easy as possible to write new and maintain existing
+security content in all the commonly used formats.
+
+### We build security content in various formats
+
+![NIST logo](docs/readme_images/nist_logo.svg "NIST logo") &nbsp; &nbsp; ![Ansible logo](docs/readme_images/ansible_logo.svg "Ansible logo") &nbsp; &nbsp; ![Bash logo](docs/readme_images/bash_logo.png "Bash logo")
+
+*"SCAP content"* refers to documents  in the *XCCDF*, *OVAL* and
+*Source DataStream* formats.  These documents can be presented
 in different forms and by different organizations to meet their security
-automation and technical implementation needs.
+automation and technical implementation needs.  For general use we
+recommend *Source DataStreams* because they contain all the data you
+need to evaluate and put machines into compliance. The datastreams are
+part of our release ZIP archives.
+
+*"Ansible content"* refers to Ansible playbooks generated from security
+profiles.  These can be used both in check-mode to evaluate compliance,
+as well as run-mode to put machines into compliance.  We publish these
+on *Ansible Galaxy* as well as in release ZIP archives.
+
+*"Bash fix files"* refers to *Bash* scripts generate from security
+profiles.  These are meant to be run on machines to put them into
+compliance.  We recommend using other formats but understand that for
+some deployment scenarios bash is the only option.
+
+### Why?
 
 This project is an attempt to allow multiple organizations to
-efficiently develop such content by avoiding redundancy, which is
-possible by taking advantage of features of the *SCAP standards*. First,
-*SCAP content* is easily transformed programmatically.  XCCDF also supports
-selection of subsets of content through a "profile" and granular adjustment
-of settings through a "refine-value."
+efficiently develop security content by avoiding redundancy, which is
+possible by taking advantage of the powerful build system of this project.
+The build system combines the easy-to-edit YAML rule files with OVAL checks,
+Ansible task snippets, Bash fixes and other files. Templating is provided
+at every step to avoid boilerplate. Security identifiers (CCE, NIST ID, STIG, ...)
+appear in all of our output formats but are all sourced from the YAML rule files.
 
-The goal of this project to enable the creation of multiple security
-baselines from a single set of high-quality SCAP content.
+We understand that depending on your organization's needs you may need
+to use a specific security content format. We let you choose.
 
-The SSG homepage is https://www.open-scap.org/security-policies/scap-security-guide/
+---
+Our OpenControl-inspired YAML rule format. Write once to build security content
```

Comment:
* The "Our OpenControl-inspired YAML rule format..." appears out of nowhere - is it one of the choices? Probably not, as there are no other choices following. If the main purpose is to show the input format, it should be clarified.
* What about linking to the OpenControl project?